blazor/speeddial/positions.md

```diff
@@ -7,11 +7,40 @@ control: SpeedDial
 documentation: ug
 ---
 
-# Positions in Blazor Speed Dial Component
+# Positions in Blazor SpeedDial Component
 
-The Speed dial Component can be positioned anywhere on the [Target](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.Buttons.SfSpeedDial.html#Syncfusion_Blazor_Buttons_SfSpeedDial_Target) using the [Position](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.Buttons.SfSpeedDial.html#Syncfusion_Blazor_Buttons_SfSpeedDial_Position) property. If the `Target` is not defined, then Speed Dial is positioned based on the browser viewport.
+This section explains the different positions of SpeedDial.
+
+## Target
+
+The [Target](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.Buttons.SfSpeedDial.html#Syncfusion_Blazor_Buttons_SfSpeedDial_Target) property in the Blazor SpeedDial component allows you to specify the element to which the SpeedDial should be attached by defining a CSS selector.
```

Comment:
The [Target](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.Buttons.SfSpeedDial.html#Syncfusion_Blazor_Buttons_SfSpeedDial_Target) property defines the selector for the element in which the speed dial will be positioned..  If the `Target` is not defined, then Speed Dial is positioned based on the browser viewport.

Reply:
Modified the content as given